pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/SchedulerManager.java

```diff
@@ -368,8 +390,13 @@ private void invokeRebalance() {
             functionRuntimeManager.processAssignment(assignment);
             // update message id associated with current view of assignments map
             lastMessageProduced = messageId;
+            // update stats
+            schedulerStats.newAssignment(assignment);
         }
-        log.info("Rebalance - Total number of new assignments computed: {}", rebalancedAssignments.size());
+
+        log.info("Rebalance summary - execution time: {} sec | stats: {}\n{}",
```

Comment:
Can we also report this stats to prometheus?

Reply:
Yes I was going to work on that in the near future.  Stats like scheduler execution time can be in prometheus.  However, a breakdown of what how many instances moved grouped by worker is not suitable to be put into prometheus